packages/thirdweb/src/react/web/ui/ConnectWallet/Blobbie.tsx

```diff
@@ -25,9 +25,14 @@ const COLOR_OPTIONS = [
  * A unique gradient avatar based on the provided address.
  * @param props The component props.
  * @param props.address The address to generate the gradient with.
- * @param props.size The size of each side of the square avatar (in pixels)
+ * @param props.style The CSS style for the component - excluding `backgroundImage`
+ * @param props.className The className for the component
  */
-export function Blobbie(props: { address: Address; size: number }) {
```

Comment:
We need to keep `size` as at least an optional prop or this is a breaking change

Reply:
I was leaning toward a breaking change because 2 reasons:
- it's easy to migrate from `size` to `width + height`
- this component is, relatively, not heavily used as compared to the ConnectButton, so a breaking change here is not too bad

that said I can still do: `style | size`

Reply:
We should avoid breaking changes at all costs, it would be a major version bump

Reply:
Yep agreed